packages/mgt-components/src/components/mgt-people-picker/mgt-people-picker.ts

```diff
@@ -1643,14 +1569,15 @@ export class MgtPeoplePicker extends MgtTemplatedComponent {
     }
 
     if (event.code === 'Tab') {
-      this.hideFlyout();
+      //   this.hideFlyout();
```

Comment:
How does the flyout get hidden now when the user tabs away?

Reply:
It doesn't. What I plan to do is this, you click on tab and the dropdown is focused, then you can navigate through it with arrow keys. When you press the tab after that, it closes the flyout.

Reply:
I think we need to be careful here with the arrow/tab behavior. The current behavior is what we landed on when working with the Viva Pulse team. The suggestions list and input field work together like a combo box. 

In short, when the input field is focused a query is run, either the zero input query or a query based on the input field. This populates the options in the list. A user can use the up/down arrows to move the pending selection and then the enter key will select the current item. The tab keystroke when the input field is selected should move to the next focusable/interactive item in the page beyond the selection list. AFAIK none of the items in the suggestions list should be focusable via tab stops.

Reply:
@gavinbarron now I see your point. So, when tabbing through the component, falling in the input element should trigger the empty/typed query and render the results in a flyout. Then clicking on the tab again, closes the flyout and onto the next focusable element. Otherwise, you can navigate through the flyout results with the arrow keys.

Reply:
Yes, that's the intended behavior